DESCRIPTION.md

```diff
@@ -10,6 +10,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 - v3.9.0(TBD)
 
+- v3.9.0(April 20,2024)
+
+   -N/A
+
+
   - Added easy logging configuration so that users can easily generate log file by setup log config in `$SNOWFLAKE_HOME/config.toml`.
```

Comment:
For other drivers, we used "Added support for log settings in a logging configuration file."